### ISSUE TITLE
culledNodes in NodeEditorGUI.cs causes NullRefException and is never used

### ISSUE THREAD

**Issue by patrickklug** [Siccity/xNode#335]
see https://github.com/Siccity/xNode/commit/f04629a120bcda8a0fcb592b152375a48e878359#r62377093

**Comment by Siccity** [comment]
Is this an issue with your pull request?

**Comment by patrickklug** [comment]
No, it's an issue in the main code base, I just commented on the exact line of code that shows the problem from the commit that added that line (via blame). If you look at the uses of culledNodes it only gets assigned to a list in the EventType.Layout phase but then gets checked for .Contains() when it's not in the layout phase. Even if that wouldn't be the case, culledNodes never gets used otherwise so probably won't do anything.

**Comment by Siccity** [comment]
oh! I see. Unfortunately I do not have time to look at it till after the holidays

**Comment by Ayfel** [comment]
Is there a fix for this?
